### PR TITLE
WT-12302 Fix compact progress time message

### PR DESCRIPTION
**Pull request by Sean04**

### CODE REVIEW

File: src/block/block_compact.c [hunk 1]
```diff
@@ -419,8 +419,9 @@ __wt_block_compact_progress(WT_SESSION_IMPL *session, WT_BLOCK *block)
     __wt_epoch(session, &cur_time);
 
     /* Log one progress message every twenty seconds. */
```
Comment (etienneptl) [inline src/block/block_compact.c:421]:
```suggestion
    /* Log one progress message periodically. */
```

**Review by etienneptl — APPROVED** [review]
LGTM % one suggestion and I will create a ticket to avoid the duplicated code.

**Review by keitharnoldsmith — APPROVED** [review]
LGTM!

**Comment by Sean04** [comment]
evergreen merge

**Pull request merged**